### ISSUE TITLE
Multiple Adjustments to Configure XDM Schema.

### ISSUE THREAD

**Issue by dhwoolsey** [AdobeDocs/platform-learn.en#32]
Issue in ./help/tutorial-web-sdk/configure-schemas.md

There are a few things on this page, so I'll bullet them out:

- On step 2 of "Creat an XDM Schema", you mention switching to a correct sandbox and then recommend that they use a development sandbox. Both of those steps wouldn't be available to non-AEP customers. You may want to drop a note outlining the difference and noting that the XDM (which is delete-able) will be created in their production sandbox. 
- The screenshot in Step 5 may be a bit dated
- Going along with the first bullet, you may want to address the over-arching issue that all of this is done in the production sandbox that they would see when they become AEP customers. We may even want to provide some kind of clean-up instructions
- A few of the steps after step 5 are different if you don't have mean AEP license. For example, you don't have the option of a schema types (time or profile based). You just add the Web SDK Schema.
- You have to click the "Add" button in the field group section to get the dialogue box mentioned in the field groups section, step 1
- You have to tab out the renaming field in order for the save button to engage.
- I really appreciate you adding the note about NOT enabling profile! 
- As mentioned above, we may want to provide some kind of delete instructions so that the production instance isn't cluttered with XDM schemas from this tutorial when customer become AEP customers.

**Comment by dwright-adobe** [comment]
* Updated the note about sandbox selection to make it clear that non-AEP customers are using "Prod"
* It looks like the most substantial change is the location of the sandbox selector. That impacts basically every screenshot in the tutorial, although i think it's easy enough to spot the new location. I will prioritize updating the screenshots where the sandbox name is highlighted.
* I think it's too abstract for people to grasp a future state where they will also have non-Prod sandboxes. The conclusion page might be a good place to include some clean up instructions. I will add a separate issue for that.
* The flow to add field groups changed a little bit--it no longer pops up the Add field groups dialog immediately after adding the schema. I adjusted the instructions. Not sure what you mean by not having the option of schema types? I see the same class options in my org without Platform.
* Updated the instructions to include hitting the "Add" button
* Good point about tabbing out of the Display name field.
* Will add delete instructions to the Conclusion page at some future point

**Comment by dhwoolsey** [comment]
Yeah....Schema types was there for me when I went back to check. I must have missed something the first time around. My mistake!
